dns-records-audit/template.yaml

```diff
@@ -65,3 +66,43 @@ Resources:
       Name: ZendeskWebhookApiDomainName
       Type: String
       Value: !Ref ZendeskWebhookApiDomainName
+
+  # Mock server records
+
+  MockServerCertificate:
+    Type: AWS::CertificateManager::Certificate
+    Condition: IsBuild
+    Properties:
+      DomainName: !Sub 'mockserver.transaction.${Environment}.account.gov.uk'
+      DomainValidationOptions:
+        - DomainName: !Sub 'mockserver.transaction.${Environment}.account.gov.uk'
+          HostedZoneId: !ImportValue MockServerPublicHostedZoneId
+      ValidationMethod: DNS
+
+  MockServerDomainName:
+    Type: AWS::ApiGateway::DomainName
+    Properties:
+      DomainName: !Sub 'mockserver.transaction.${Environment}.account.gov.uk'
+      EndpointConfiguration:
+        Types:
+          - REGIONAL
+      RegionalCertificateArn: !Ref MockServerCertificate
+      SecurityPolicy: TLS_1_2
+
+  MockServerCertificateAliasRecord:
```

Comment:
Might want to consider another name here! As this isn't the record for the certificate, it's for the mock server's domain name.